.npmignore

```diff
@@ -10,3 +11,8 @@ coverage
 
 # github
 .github
+
+.sass-lint.yml
+/*.json
+/webpack.*.js
+/.*
```

Comment:
This is fine, but at some point we may want to whitelist via the `files` property in `package.json`. https://docs.npmjs.com/cli/v7/configuring-npm/package-json#files